.github/workflows/android-develop-pr-workflow.yml

```diff
@@ -52,8 +52,11 @@ jobs:
       - name: Change wrapper permissions
         run: chmod +x ./gradlew
 
-      - name: clean build
-        run: ./gradlew clean build
+      - name: spotless check
+        run: ./gradlew spotlessCheck
+
+      - name: generate debug apk
+        run: ./gradlew assembleDebug
```

Comment:
I'd suggest to remove the mock build from pipeline, and build all other envs together:
`./gradlew clean assembleDev assembleStage assembleProd -x test --stacktrace`

Reply:
updated